packages/react/src/components/Table/TablePropTypes.js

```diff
@@ -125,6 +125,7 @@ export const TableColumnsPropTypes = PropTypes.arrayOf(
       placeholderText: PropTypes.string,
       /** if isMultiselect is true, the table is filtered based on a multiselect */
       isMultiselect: PropTypes.bool,
+      customInput: PropTypes.elementType,
```

Comment:
Please add a large comment describing what the expectation of this customInput will be.. eg,

```
/**
 * customInput should be a React component that will accept the following props.  value, onChange, (list all props we are sending)
 * The expectation is that the input component will use value as the default value and onChange will be called when the value changes.  The FilterTableRow will debounce the values so so the component does not need to do this
 */
```